### PR TITLE
Don't fetch wiki id for files that are on the wiki that this requests…

### PR DESCRIPTION
**Pull request by hakubo**
… is being called on

### CODE REVIEW

File: extensions/wikia/WikiaHubsServices/models/EditHubModel.class.php [hunk 1]
```diff
@@ -614,11 +614,20 @@ private function getFileUrl($fileName, File $file) {
...
-		if ( !empty ( $wiki ) ) {
-			return $wiki->city_id;
+		$repo = $file->getRepo();
+
+		/**
+		 * If file is from WikiaLocalRepo then city id
+		 * will equal to wiki from current context
+		 *
+		 */
+		if ( !( $repo instanceof WikiaLocalRepo ) ) {
```
Comment (michalroszka) [inline extensions/wikia/WikiaHubsServices/models/EditHubModel.class.php]: Note, that `getDBName()` only exists in `extensions/wikia/VideoHandlers/filerepo/WikiaForeignDBViaLBRepo.class.php`. This check therefore has a leak.

**Comment by bognix** [comment]
ping @kvas-damian @michalroszka

**Comment by kvas-damian** [comment]
LGTM

**Comment by michalroszka** [comment]
:+1:

**Pull request merged**